int32 id
string ros_package
string executable_name
string[] args
string state
